app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_by_name = []
+
+    for person in people:
+        person_by_name = Person(person["name"], person["age"])
+        people_by_name.append(person_by_name)
+
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = (
+                Person.people)[person["wife"]]
+        if person.get("husband"):
+            Person.people[person["name"]].husband = (
+                Person.people)[person["husband"]]
```

Comment:
bad formmating here (
please, use a code style as mentioned in this file:
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md

Reply:
There are not examples with parentheses on the URL, if you are talking about them, I'll change now.

Reply:
`Person.people[`
`   person["name"]`
`].husband = Person.people[person["husband"]]`
f.e. u could break the line like this
or like this:
`Person.people[ person["name"]].husband = Person.people[`
`    person["husband"]`
`]`